common/macresman.h

```diff
@@ -52,6 +56,68 @@ namespace Common {
 typedef Array<uint16> MacResIDArray;
 typedef Array<uint32> MacResTagArray;
 
+/**
+ * Class containing the raw data bytes for a Macintosh Finder Info data block.
+ */
+struct MacFinderInfoData {
```

Comment:
What is the purpose of these? E.g. you could just define `const kMacFinderInfoData = 16` and use the very similar code.

Though, it is up to an individual taste.